src/main/java/HttpGetRequest.java

```diff
@@ -0,0 +1,40 @@
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+class HttpGetRequest extends UserInterface{
+    StringBuffer response = new StringBuffer();
```

Comment:
I suggest renaming "response" to  "mResponse". I think it will be more readable variable for your project.